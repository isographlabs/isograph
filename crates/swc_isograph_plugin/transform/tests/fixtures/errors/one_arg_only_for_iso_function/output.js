export const HomeRoute = iso(`
  field Query.HomeRoute @component {
    pets {
      id
      PetSummaryCard
    }
  }
`)();
  
