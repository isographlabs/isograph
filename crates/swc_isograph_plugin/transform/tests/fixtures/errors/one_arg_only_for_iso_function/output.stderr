
  x Invalid iso tag usage. The iso function should be passed at most one argument.
   ,-[input.js:1:1]
 1 | ,-> export const HomeRoute = iso(`
 2 | |     field Query.HomeRoute @component {
 3 | |       pets {
 4 | |         id
 5 | |         PetSummaryCard
 6 | |       }
 7 | |     }
 8 | `-> `)();
   `----
